synfig-studio/src/gui/docks/dock_info.cpp

```diff
@@ -197,6 +203,11 @@ studio::Dock_Info::~Dock_Info()
 {
 }
 
+void studio::Dock_Info::on_stop_button_clicked()
+{
+	async_renderer->stop();
```

Comment:
Two things related to stop button:
- check if async_renderer is valid before defer it (`if (async_renderer)`)
- it should be sensitive (= enabled) only when something is being rendered :)

Reply:
Committed the changes!
I've moved the button under the progress bar to keep the symmetry of the dialog:
![button_placement](https://user-images.githubusercontent.com/6705690/88479514-ed5ecb00-cf4f-11ea-8398-da3898a649a3.png)